src/app/_utils/sortEntriesByDate.ts

```diff
@@ -22,14 +22,29 @@ export function sortEntriesByDate<Entry extends SortableByDate>({
   sortOption,
 }: SortEntriesByDateParams<Entry>) {
   return [...entries].sort((a, b) => {
-    const dateA = validateDate(a[sortBy])
-    const dateB = validateDate(b[sortBy])
+    const dateA = validateDate(
+      typeof a[sortBy] === 'undefined'
+        ? ''
+        : a[sortBy] instanceof Date
+          ? a[sortBy].toISOString()
+          : a[sortBy],
+    )
+    const dateB = validateDate(
+      typeof b[sortBy] === 'undefined'
+        ? ''
+        : b[sortBy] instanceof Date
+          ? b[sortBy].toISOString()
+          : b[sortBy],
+    )
```

Comment:
`validateDate` is already checking if the argument is falsy, so no need for the double ternary operator check?

```suggestion
    const dateA = validateDate(
      a[sortBy] instanceof Date ? a[sortBy].toISOString() : a[sortBy],
    )
    const dateB = validateDate(
      b[sortBy] instanceof Date ? b[sortBy].toISOString() : b[sortBy],
    )
```